packages/core/src/header-builder/authorization-header.ts

```diff
@@ -13,7 +13,7 @@ import {
   sanitizeDestination
 } from '../scp-cf';
 import { ODataRequestConfig } from '../odata/common/request';
-import { ODataRequest } from '../odata/common/request/odata-request';
+import type { ODataRequest } from '../odata/common/request/odata-request';
```

Comment:
👍 
In terms of the task you're working, whether this helps reducing the circular dependencies that detected by Madge?

Reply:
sorry this question I do not get.

Reply:
clarified via call.